include/RAJA/exec-cuda/MemUtils_CUDA.hxx

```diff
@@ -127,37 +352,35 @@ void freeCudaReductionTallyBlock();
 /*!
  ******************************************************************************
  *
- * \brief  Return pointers into shared memory blocks for RAJA-CUDA reduction
- *         with given id.
+ * \brief  Get device memory block for RAJA-CUDA reduction variable  with
+ *         given id.
  *
  *         Allocates data block if it isn't allocated already.
  *
- * NOTE: Block size will be:
+ * \param[out] device_memblock Pointer to device memory block.
  *
- *          sizeof(CudaReductionBlockDataType) *
- *            RAJA_MAX_REDUCE_VARS * ( RAJA_CUDA_REDUCE_BLOCK_LENGTH + 1 + 1 )
+ * NOTE: Total Block size will be:
  *
- *       For each reducer object, we want a chunk of managed memory that
+ *          sizeof(CudaReductionDummyDataType) *
+ *            RAJA_MAX_REDUCE_VARS * RAJA_CUDA_REDUCE_BLOCK_LENGTH
+ *
+ *       For each reducer object, we want a chunk of device memory that
  *       holds RAJA_CUDA_REDUCE_BLOCK_LENGTH slots for the reduction
- *       value for each thread, a single slot for the global reduced value
- *       across grid blocks, and a single slot for the max grid size
+ *       value for each thread block.
  *
  ******************************************************************************
  */
-CudaReductionBlockDataType* getCudaReductionMemBlock(int id);
-///
-CudaReductionLocBlockDataType* getCudaReductionLocMemBlock(int id);
+
```

Comment:
Remove blank line so doxygen knows that the comment applies to the method immediately after it.